pyanaconda/simpleconfig.py

```diff
@@ -202,7 +202,7 @@ def __str__(self):
 
         return s
 
-
+# TODO: this function is never used
```

Comment:
Remove

Reply:
Do we really want to do that ? If we remove the __str__() method, the simpleconfig class instances will not have a string representation. That could be inconvenient during debugging & for python-meh dumps.

Reply:
It is not the `__str__` method, but `simple_replace`.

Reply:
Removed `simple_replace`.